.gitignore

```diff
@@ -0,0 +1,13 @@
+**/__pycache__
+env/
+venv/
+.env
+.idea
+.pytest_cache/
+.mypy_cache/
+bin/
+lib/
+lib64
+**/$*.py
```

Comment:
Out of curiosity, what is this?

Reply:
ohhhh so soryyy, take another one from another project, just cleaning it O_O. Sometimes speed is not a good friend :)

Reply:
No, but what IDE/editor/thing creates files named like this?  I'm curious.